### ISSUE TITLE
Feature Request: Armory Superviser, Arms Master, or some other job title I don't know

### ISSUE THREAD

**Issue by FudgePucker** [d3athrow/vgstation13#7415]
We have a problem, Science can produce weapons that are better than any weapon in the armory, then just quickly popping open the lockbox and arming up more than the officers who have to beg the warden or HoP for anything more than their punny taser.

Science can produce these weapons intentionally to help arm security, but security officers can't open them, only the warden and the HoS can. The warden can't leave brig at all, if he does then someone can hack into armory while he isn't looking, or murder him and take his ID to get into the armory. The HoS is busy. And science isn't going to give security shit because they are just graytiders who have access to guns.

SO HOW DO WE SOLVE THIS?
A new job, this new job would be located in an office connected to armory.
He would have a modified protolathe and a fancy workbench (that I will cover later).
This jobs protolathe can only produce stock parts (reason I will cover later) and weapons. Most weapons besides temp guns and some other sciency weapons (rapid fire syringe guns and that fancy potato gun that can crit in one hit), this new protolathe will be the only one being able to make AEGs.

The warden can keep in his office, managing the brig, the HoS can help keep the station afloat, this new job is arming the officers for the situation at hand.

BUT WAIT isn't this just mechanic science flavor? Whats stopping them from just mass producing weapons? They are a part of security and are expected to not be a shit or get security banned. Handing out AEGs as science has zero risks from admin intervention.

BUT WAIT won't this cause another armory ID for someone to loot off of a corpse? I had an idea to get past this, a secruity dongle that acts like an armory ID, kept in his office in a safe. Carrying it around on you and getting stripped and looted is sec bannable. You are not the HoS, you are not an officer.

Because their only job is to make sure security officers are well stocked, they can go out and wait for supplies.  They can sit in medbay and wait for some pill bottles of meds  to be produced for officers, can wait on cargo for shipments, and help get supplies.

<h1>Shit is about to get crazy</h1>

To help make this more unique of a job instead of being just being mechanic security flavor, someone is going to have to ~~do some major programming that I have no idea how to pull off~~ make some better gun datums.

That workbench in their office can upgrade non protolathe weapons. You can make super recharging batteries for eguns. You can shorten the barrel of an ion rifle  to make an ion rifle that can fit in a backpack, only for less shots and only 1x1 blast instead of 3x3 blast.
Just add in new stock parts into a gun to help improve them (or add new parts entirely).

BUT WAIT why would miners bring mats to security?
They can upgrade the kinetic accelerators to be recharging, higher damage, and more.

So in conclusion, this would be a job that would remove some power from research (I don't think anyone thinks research is fucking balanced.) and give more fire power to a department that is the most shit on. It would also add in new unique gun mechanics not seen on any other server.

**Comment by icantthinkofanameritenow** [comment]
I support this. It's about time Science and Cargo were knocked down a peg, and the people whining about MUH SPEHSS 2ND AMENDMENT can just go and ask this guy.

**Comment by ghost** [comment]
Fuck no. It's another dupe rnd that has zero reason to exist and little to nothing to do other than be red tide with +access. Splitting minerals is just mechanic syndrome 2: The phantom job from the pre-plastic stock parts days. So if you're having to rob rnd to essentially use rnd, you are rnd and there's no reason for a shitty knockoff to exist. The warden already has the job of supporting sec. It's his job to run the brig, prisoners, and order more stock. That he can't talk to other crewman to order supplies through a pda or supply request console or is too busy validhunting as red tide with armory access is a problem with the player, not the role or the need for a new copy of the role.

Security can and always had the freedom to get guns at cargo. To triple IED+baton into, flashbang and buy guns when cargo is being shit. They can but rarely ever take the initiative to ask rnd for guns. Often cause they're busy but I feel safe saying my PDA isn't regularly blowing up from the warden, HoS, or even a powergaming cop asking me to get some guns ready for him then deliver to the brig in a locked locker or pda him for a hot pickup. Even when I campaigned for constant console locking, I constantly had to re-lock the console, nothing is stopping you from going in with the all access you should have and print some or to rotate windows in and fix on the way out. If sci is acting up at epidemic levels, which would be an IC issue, you can get the band together, get comms turned off, then bust the whole dept. Pull the circuits, short the APC's, bolt the doors. Officers and detective are just as capable of opening lockboxes as every other nigger who does so and crying that others are better armed than you just shows you're a pussy ass faggot. Especially when flashbangs are one of the strongest weapons in the game. 

You can get crates at cargo then go make a fleet of securitrons/ED-209's and start random searching cult style with them gunning and batoning down anyone who pulls heat.

If you want to go full tryhard, you can run a confiscation purge. Follow S.O.P., go code red, declare station wide martial law and confinement to dept, and brig anyone out of dept for any reason until you've finished your sweep. You can do less and just ball up, random search, permabrig anyone who uses illegal guns to shoot at you, long brig, confiscate and demote the rest, then if you get bwoinked have them banned for being a greytiding dindu and banbaiting. I don't know why you'd bother to tryhard cause those players don't want to die either so when somebody starts murderboning they're gonna help kill the murderboner but whatever. If you want to crush you balls in a vice that's your thing as sec and space law has your back.

So It's simple, stop being a fucking autist that doesn't ask cause abloo bloo bloo a guy who was grudging sec was a meanie once and didn't listen so I'm never ever gonna try cause everyone in sci every round is always a big bad meanie even when maybe one of them this round would if I just asked. All you faggots that bitch about science are like that. Just cause of bad rounds you stop trying then you continue to cry despite not engaging with us so no shit you didn't get stuff you never asked for from busy people who are doing multiple jobs. Just ask everyone in the dept once or twice to coordinate with each other to get the order together and you'll get lucky more than you think. Especially if you can have someone run metal for you if you're warden or yourself if you're a cop/detective so you've paid your way for it.

**Comment by icantthinkofanameritenow** [comment]
@SarahJohnson489

And how do you propose dealing with them when they refuse to make weapons for security and nobody else? You can't raid them since they have better guns than you, and the RD is either a figurehead, deceased, or actively helping them in violating space law.  Plus, they give guns to EVERYONE- including antagonists without realizing it. 

They simply cannot be trusted to make their own weapons. And for the record, flashbang protection is as simple as making a welding helmet and using the earmuffs that they just so happen to have on the research outpost. 

And it isn't "one bad round", its a ceaseless trend that has to stop, and this is coming from someone who MAINS science. So tell me, how would you control who gets guns when the honor system we have now fails over and over again because those  Scientists care more about their valids than the law?

**Comment by icantthinkofanameritenow** [comment]
And I do ask, and they're happy to give then to me. They also give them to the Clown, random assistants, and of course to themselves. Ifnthry can't learn some fucking self-control,  they deserve to lose their gun privileges. 

Oh, and literally all of what you suggest gets you instantly branded as shitcurity, which is why I have to make an issue like this instead of resolving it IC (which does nothing and leaves the real problem to fester). 

I have to wonder if you have a vested interest in keeping Security deliberately weak.

**Comment by ghost** [comment]
Ah, so it's clear you're just shit at sec and crying then. They can't lethal you without provocation without being banned or a traitor which leaves you the initiative to spray water and flashbang the place up. Their "better" guns don't amount to jack shit. That they have guns is unauthorized combat gear construction which allows you to bend them over to the point of buying lasers from cargo or rotating in when they're elsewhere to get and pop your own then kill them so you can get the shit off them cause they wont stop shooting up the place.

If you want to play the metagame, it doesn't matter than they give an antag guns cause 99 times out of a hundred, you can just ask or rotate in and print 5 assault rifles at rnd, mechanics, or elsewhere. If they want research guns, they're gonna get them. The difference is that they handed out so much 3 assistants, a mime and a chaplain also have guns so when the killing starts there's that many more armed crew who are gonna kill him cause they don't want to get killed. As a further note, it's not just a sci problem, they can get buckets of guns at cargo as well but sci guns is the hip new thing for bad sec to cry about.

Earmuffs don't give you full and in most spots it's not like you can't get out of range and back in time to taser(Edit:brain fart) them before they get back up.

You can enforce space law violations, raid, and remove protolathes and printers to security to have requests ignored by the warden. This is a player problem, not a code or a job problem. The same as when it was cargonia every other shift where we raided the fuck out of them, permabrigged them, and the ones using lethals to defend their greytiding got banned.

You can stop being such shitcurity that people are willing to risk bans to attack you

I often play HoS and officer in lowpop. I often have a shotgun/combat shotgun full of lethals, an ablative, magboots, multiple flashbangs, a pepperspray of water, 2 lasguns as officer/detective until I get AEG's after I see a mineral drop in sci or 2 eguns as HoS and pulling a locker or beepsky. I guaranfuckingtee you I'm not outgunned by sci any day of the week.You're only as "weak" as you are ignorant and lazy. 

Ignore it and let validhunters valid the murderbones, 
Perma+implant people abusing them as it happens and let the rest hang onto their guns for when murderbone bones for them
Bag and brig both the provider and the requester and demote them for combat gear construction
Just raid the dept and impound all the protolathes and imprinters like we did all the time for cargonia niggers. I'd recommend constant console locking but good luck getting it to stick, I tried for a month.

**Comment by icantthinkofanameritenow** [comment]
And you sound like an arrogant cunt unwilling to accept that you'll be BWOINKed incessantly because for some inexplicable reason RnD gets special admin protection. The Adminbus coddles them to the point a raid on them is a guaranteed jobnam for being "shitcurity". Apparently the mere act of raiding counts as " provocation" now. 

A little fucking BALANCE is all I and so many other people ask. At least make it so they can't make AEGS less than half an hour in, for fuck's  sake.

**Comment by ghost** [comment]
I don't even know what you're trying to argue at this point. It's an IC issue, if they're breaking the law and you give a shit go bust them.(Though it just makes shit harder for you but whatever bro) If a trialmin cops an attitude it wont stand up in an appeal. If sci is killbaiting with its gunshit and the admins don't care you can make a ban request for a headmin to kick their ass up around their ears.

Logs or go fuck yourself with your continued boogeymans. I'm getting fucking sick of them and everyone who brings them up any time in defense of security. Don't kill people you could have just safely cuffed who weren't using lethal force or stuns+intent to kill and have a space law reason to be untouchable as security.

Cargo can get combat shotguns in 4 minutes, sit by a locker disposal then double headshot you. Whoop de fucking do. chem can make 2 1s explosion grenades, flick their magboots, then double tap you. Or hell just use pacid. Whoop de fucking do. Eng can rush one-tank bombs and use their magboots plus ZAS to double tap you. Whoop de fucking do. You can get your own guns and shields just like anyone else. N2O cans are 1-2 doors away from anyone with a toolbox and if that's too obvious you can just load it into N2O bombs to knock out lazy sec with a word. You can shut off a hallway's CO2 scrubbers and flood it, feed them groans sleepy and drag people into maint when they pass out. There's no balance in the game, only taking the initiative(and abusing flashbangs/slip strips). Sec can do their job or quit and go to tg and cry for the big manly coders to code you some crutches.

**Comment by FudgePucker** [comment]
So your entire argument boils down to 'git gud'?

I just think that a department that can produce AI upload boards shouldn't also be able to produce guns that are better than anything in the armory.

**Comment by ghost** [comment]
My argument is that it doesn't matter and maybe you should either fight smart if you can't hit a stationary target using a console or in a confined space or leave it be instead of abloobloblo coders save meeeeeee. You can make some of them elsewhere. You can get comparable weapons elsewhere, with less effort, and faster.. And on top of all of that you have the advantage of the initiative to flashbang and/or slip strip the shit out of them. They can't use half the advantages the guns they make have. So handle it like the IC issue it is. Let the validhunters kill the murderboners, raid them, or raid them and condemn the dept then get searching. Like the cargonia of old, get the faggots banned and the ones who weren't get to deal with their sentence and demotion then perma if they keep stirring shit.

**Comment by FudgePucker** [comment]
Admins don't give a shit about security most of the time.
Did I ever tell you how I was shoved into the telescience chamber then gassed with plasma?
Three scientists including the RD just watched this.

You want to know something funny?
None of them got banned.
I got a note for gunning down the RD after he watched me get murdered.

Fuck research having guns.

This would still allow the crew to be armed in the situation of an emergency instead of having some shitty firing pin system, its just they have to be produced by the armory instead of in science.

This job wouldn't have the same problem of getting mats that mechanics have because they would be able to incentivize miners with better pokeman hunting gear, mechanics are just research, only empty most of the time.

Science mass producing guns isn't something that only happens once every blue moon, it happens EVERY ROUND. EVERY ROUND THEY PRODUCE AEGs.

**Comment by ghost** [comment]
You can't kill for not helping you, especially as security, no fucking shit you got a note you psycho. What you can do is use logs and pictures and make a ban request for the guy that put you in there instead of crying for a coding change.And gee, now it all makes sense, you're salty as fuck and want a coding change cause you got your ass kicked. 

So your solution is to have a job whose sole reason to exist is to be not-rnd in security. Even if you just passed it off onto the warden's responsibilities it's still gonna have the same problem as pre-plastic stock parts mechanics had which was is was absolutely crippled. Especially in med/low pop when there flat out were none. You can try to get some but good luck as minor gun upgrades aren't a diamond drill and with how poorly mechanics has been treated outside of a few good frequent shaft miners, it'd get the shaft all the same and spend most of its time in rnd or robbing rnd to go be an rnd clone. This job would be a sick fucking joke. 

Yes, yes they do make guns every shift. And it's rarely a problem at all. And if you decide you want to tackle it, it's nothing you can't tackle with a flashbang and some water. You can do your job and enforce space law with raids and impounding machines or get over it and keep making ahelps whenever they kill someone with it then ban requests if you don't agree with the admin decision. The coders shouldn't be doing your job.

**Comment by icantthinkofanameritenow** [comment]
I'm copypasting something from the sister issue, because it's saying it in a way better than I can. Maybe it'll get it  through your thick fucking skull, Sarah.

Alright, time for some walls of text if you're posting your own. Remember what the point of this is to nerf the crew because they are way too strong in relation to Security and to give some sort of incentive for people to play Security.

It's not hard to fuck over Security and get away with it. You don't have to kill them. You just have to stun, cuff, detain in Science on a chair. Or not even detain. Just strip them naked and throw them in disposals. None of that shit is bannable, but it still completely fucks you over. I'm not going to talk about MUH ADMINS, but I will say that it's not hard to just claim escalation and an admin will just forget about it. You keep going on and on about these dumb power gaming strategies you would do as HoS or Security to stop it, but I'll be honest dude, I saw you as Crunch Buttsteak and you were a really inefficient HoS. That's beside the point though. I mentioned it in the thread where I was arguing with this guy who created the issue, evidently I convinced him of something so I'll just copy paste the relevant parts. Or you can just read the conversation here. https://4ch.be/vg/thread/126852604/#126856517

For all the counters you keep posting I can only say this,

Yes, there are counters for all of this, but even with these counters, there are counters that the crew can take for those counters and it's never as simple as, do this if they do that. You are always assuming a pristine condition in which everything is so simple to accomplish or even possible. It's not always the case that you have an AI who is going to be helpful. It's not the case that you are going to be free or have enough people to raid Cargo, Science, or whoever is acting up. You simply can't be in all places at all times. You have to prioritize. Some things are going to slip through the cracks, it's just that on /vg/, most things will slip through the cracks. The times I played Security on /vg/ (where I was the HoS as only Security aside from maybe the detective and a borg) I was forced to ignore all crimes and just get the antagonists. If I didn't I became swarmed with tasks that took way too much time and the traitors went around killing people and eventually blindsided me while I was brigging the guy who kidnapped the clown.

The point again, is that the situation for Security on /vg/ is abysmal. You are outgunned and outnumbered most of the time. Not many people can handle it, and so not many people play it. Especially new people.

Now back to the rest: If you're outnumbered 3 to 1 or even 5 to 2, it doesn't fucking matter that you can flash bang a guy (even though he could have flash bang protection) because you can't cuff 3 people at once and as soon as they get up you have to either chain stun them very accurately to cuff the rest (let's hope to god all people are in one room and nobody just wanders in to shoot you) and then cart them off one by one.

About the lockbox thing, I completely disagree with this guy. Removing the lockboxes entirely is fine IF AND ONLY IF the firing pin system is implemented.

I'll say again my solutions to the problems. I don't know why I'm bothering as nobody will take it seriously.

Firing pins. Traitors can emag the unpinned gun to make it work. Firing pins are only in the armory. If you want to spam guns fine, but you have to break into the armory first.
Security cargo crates (GUNS) only able to be ordered with a Security ID in the console.
I should re-iterate I'm not some kind of crap Security player who whines and can't get shit done. I was literally stickybanned from /tg/ for apparently ruining the rounds as HoS by incinerating all the rounds traitors every single time, they called me "JLP but worse". It's an issue. Maybe not for YOU or for ME, the superman who has played the game for years, but for most people it is.

**Comment by ghost** [comment]
I agree whole heartedly that sec shouldn't have to deal with greytiding but it's the reality of having freedom on a light rp server. So do your job and break your foot off in their ass one raid at a time until you're getting it right more often than not. Keep impounding those lathes and printers. Break the metagame of sci/mechanics can get away with guns by attemtping to cram them in brig like sardines in a can every single time until you regularly succeed and they're so sick of having to wait out the round or quit that they don't feel it's worth it to be irresponsible with them. Heck, once they start pre-escalating you'll be knocking a bunch out with jobbans. Crying for a coding change instead of doing your job is disgusting. Especially a hollow shell of a job and fucking everybody in the ass when the system breaks down like it always does be it from lowpop hell or people getting whacked. I know how fucking retarded the suggestions from you two are. I know how badly it's gonna fuck shit up but neither of you give a fucking damn about the fallout. Even if this is done something else will start kicking your ass and we'll be going over it all over again. It's degenerate is what it is.

**Comment by icantthinkofanameritenow** [comment]
You keep talking about "raids", but not even once have I ever seen one pulled off successfully, least of all by YOU. 

So put your money where your bug mouth is and prove its even possible before you take the boiler than thou "git gud" approach you keep boasting about.

**Comment by ghost** [comment]
Cool beans, hang out for a cult round where I can be bothered to give a shit about guns that are otherwise by and large used to kill people killing all of us.

**Comment by icantthinkofanameritenow** [comment]
We both know a cult that goes loud like that is a dead cult, atop moving the goalposts. Especially when science is almost always the very first department to get culted specifically BECAUSE they can mass produced guns.

**Comment by ghost** [comment]
And we both know, someone who shouldn't have a gun drawing it for no reason is trying to jump you
We both know once one fucks up with his stuff on him and you know it's cult, it's time for dept raids.
We both know flashbangs, water, and having the initiative absolutely ravages their asshole.

Yes you shouldn't have to deal with greytiding shitbags, but that's a problem with the greytiding shitbags and having freedom instead of having the coders and admins be security for IC matters. Punish the fuck out of them until they don't feel like doing it instead of making the game worse when shit's tumbling down. Exactly when sec most often fumbles the ball be it from being too busy, too few, or already dead.

**Comment by icantthinkofanameritenow** [comment]
And how do you fix the "too few" part beyond making the security experience less soul-crushing? Anyone would cry foul if I started press-ganging people into it.

And we already discussed that water and flashbang are not and will never be the panacea you claim they are. As I said- try doing a raid yourself RIGHT NOW and perhaps you'll appreciate why this is an issue to begin with. 

You won't last ten seconds, I guarantee it.

**Comment by ghost** [comment]
You just change your win state to doing better than you did last time and do the best you can with what you got. A beepsky, going full powergamer and zero chances helps when you're flying solo. Someone's gotta be the first man to fix bayonet and charge into no-man's land. To help draw the more timid sec players. The admins have been doing a much better job of grilling the ahelper before bothering you and if you've even loosely been following space law you're golden. It's when the officer can't control their murderboner or they're being excessive that it catches up to them. Also eat your damn food. that slow catches up with you at the worst times if you don't

Working in a team  that's always in sight of each other and says where to go massively raises your survival chances.

It's not a panacea but it can easily make things a damn lot easier. I see it too rarely on both sides of the fence.

I don't know who you play or how much you play but I've been doing sec off and on for a while now. I haven't had a problem with all the guns zipping about and in cult when it's been raid time it's by and large worked out. If you care about the gunshit in regular rounds just keep at it, remember your flashbangs, your water, lead your target and keep at it until you can get it done. Heck take a locker if you're worried about dumbshots from offscreen strafing the hall.

**Comment by icantthinkofanameritenow** [comment]
I don't think you understand. You literally cannot raid when you're the ONLY Security staff on the station. Even Beepsky can't stuncuff everyone at once and it'll usually get scrapped too. Plus, I can count on one hand how many people can use a flashbang without it stunning themselves in the process. Sure those sunglasses help, but it still knocks you over long enough to be fucked up yourself- and from there it's into disposals you go.

It isn't about buffing Sec, its about nerfing the crew.

**Comment by FudgePucker** [comment]
So I should have to raid two entire departments every single round?

Seriously, this idea is the middle ground, having firing pins across the station from the guns is a shitty idea for blob rounds and other emergencies, but why have science hand them out instead of security?

The reason why mechanics don't get mats is because they are empty rooms every other shift, and produce the same shit that science does. This new job provides an incentive to miners to provide some mats by upgrading their gun, so instead of carrying a belt full and tossing them aside in the middle of a life or death situation, they can carry one badass gun that can fuck up mobs in two shots.

**Comment by ghost** [comment]
Devil's in the details of your boogeyman, long as they're not a taser duration apart you can drag em together and a batonlock em to cuff them all until the cows come home. Distance between them and help or them being clustered up to get one already cuffed. You can lockerweld one to get him out of the way for a minute, you can blindside with an N2O can. They're all impractical and so is the assumption that the entire dept is a security seeking missile. Heck, if you're that worried just dope up after getting an ablative, riot, and locker shield then go kick ass if you need to be the one-man judge dredd. Frequently butting heads with sci will give them lots of practice and lots of time to be salty. If you don't know or don't want to be shackled to your pill bottle you can rustle up help. you can subvert the silicons. There's a bazillion and one things you can learn that this time might make the difference. Stop crying for papa coder to do your job for you and do it or stop murdering everything that moves or whatever the hell you guys are doing that's bad enough to get people shooting you up. It's fucking cargonia round 2 boogaloo, deal with it. It's not hard and I'm sick of this circular argument where we're getting nowhere.

Fudge, we don't need a dupe job that is a warden, but not and is rnd, but not, and is just gonna break the law you care oh so much about to get minerals, 30-50 minutes later of fucking nothing, cause the miner just wants his drill and can't be fucked to split. I'm sorry but I'm not articulate enough to express the magnitude of how fucking retarded the job is. You can ask sci for guns. If you're buttflustered about gun ownership that's a nonissue you can enforce space law erryday until it's clear that the days of gun tolerance are gone and to challenge that is to forfeit your round. 
Cached firing pins are gonna get meta'd however it takes, which makes them existing pointless like lockboxes, unless it's impossible which gets back to hard checks in high tension emergencies resulting in the ball getting fabled through being busy, missing the bajillion messages, being dead, or a million other things. Safes can be reset. You say a puny cop or detective can't get heat but they can get plenty of heat from plenty that is quite sufficient when 99% of the people you're trying to go bag can't double burst through the glass so so what they have bullets they can't use and their guns recharge when you just flashbang/slip or god forbid actually hit your target and that you don't know that pisses me the fuck off. Go arrest the fucks, arrest the fucks that try and help then and keep doing it. Ahelp the murderers instead of killing bystanders like was done in non-lethal cargonia raids.  Learn from your fuckups and keep at it. This just whole fucking thing is so goddamn retarded and i'm platinum mad.

**Comment by icantthinkofanameritenow** [comment]
@SarahJohnson489 
It shows. But all of what you said assumes Security is several times more robust than it is in the average round, whereas the majority of time it's one guy who doesn't have armory access. 

Think about that for a second. Shouldn't the playing field be even slightly more level than that?

**Comment by Probe1** [comment]
The discussion ran its course and we got make shift guns which is some change.
